src/wdio/services/TerraService.js

```diff
@@ -67,17 +67,17 @@ export default class TerraService {
 
   // To more passively support code splitting in terra dev site, wait for data to load before progressing with the test.
   // eslint-disable-next-line class-methods-use-this
-  afterCommand(commandName) {
-    if (commandName === 'refresh' || (commandName === 'url')) {
-      if (global.browser.isExisting('[data-terra-dev-site-loading]')) {
-        try {
+  afterCommand(commandName, args, result, error) {
+    if ((commandName === 'refresh' || commandName === 'url') && result === 0 && !error) {
```

Comment:
If an wdio error is passed, should we throw it?

Reply:
I assume wdio will throw that error itself and we don't have to handle it here.